tr/Cargo.toml

```diff
@@ -9,19 +9,26 @@ repository = "https://github.com/woboq/tr"
 documentation = "https://docs.rs/tr"
 keywords = ["internationalization", "translation", "l10n", "i18n", "gettext"]
 categories = ["internationalization", "localization"]
-edition = "2018"
+edition = "2021"
+rust-version = "1.81"
```

Comment:
Why is the MSRV so high?

Reply:
From the last failed build attempt in the CI:

```
error: package `half v2.6.0` cannot be built because it requires rustc 1.81 or newer, while the currently active rustc version is 1.74.1
```